app/views/pages/about.html.erb

```diff
@@ -0,0 +1,27 @@
+<div class="container release-notes col-md-12">
+  <h1 class="page-heading">About the New Catalog</h1>
+
+  <p>Initially released on 28 April 2016, this catalog is the result of a nearly two-year-long project to replace our twenty-year-old <%= t('blacklight.voyager') %> and incorporate the best features of contemporary library search technology.</p>
+
+  <p>The catalog is based on <a href="http://projectblacklight.org/" target="_blank">Blacklight</a>, which is an open source collaboration across several institutions including
+    <%= link_to("https://newcatalog.library.cornell.edu/", target: "_blank") { "Cornell" }  %>,
+    <%= link_to("https://catalyst.library.jhu.edu/", target: "_blank") { "Johns Hopkins" }  %>,
+    <%= link_to("https://searchworks.stanford.edu/", target: "_blank") { "Stanford" }  %>,
+    <%= link_to("http://search.library.yale.edu/", target: "_blank") { "Yale" }  %>, and
+    <%= link_to("http://projectblacklight.org/#examples", target: "_blank") { "a number of others" } %>.
+
+  <p>As the examples above demonstrate, Blacklight is highly configurable and customizable, which provides a number of advantages:</p>
+
+  <ul>
+    <li><strong>Tighter Integration with Existing Services.</strong> We run less risk of having to rely on third parties for intergration with the several different requesting and fulfillment options that the library has to offer.</li>
+    <li><strong>Control over Look and Feel.</strong> We have complete contol over page layouts and how all aspects of the application are rendered. This allows us to create a more seamless experience for users as they pass between the catalog and library website, as well as ensure that the catalog is accessible and usable on mobile devices.</li>
+    <li><strong>Custom Relevancy.</strong> We know our collections and users better than anyone else, and the search engine beneath Blacklight gives us the power to tune search relvancy to best suit our environment.</li>
+    <li><strong>Community Development.</strong> As open source software developed by the academic and cultural heritage community, Blacklight allows us to leverage the talents of our own staff as well as that of dozens of software developers and librarians from around the world.</li>
+    <li><strong>New Features.</strong> The aformentioned community is always adding new features to Blacklight, and when a feature does not exist in the application, we are free to build it rather than having to wait for a vendor to do it for us.</li>
```

Comment:
The aforementioned community is always adding new features to Blacklight. When a feature does not exist in the application, we are free to build it without having to wait for a vendor to do it for us.  (fixed a typo in 'aformentioned' and verb tenses were adjusted to be in sync)